src/Microsoft.ML.Core/Data/DataKind.cs

```diff
@@ -55,50 +95,67 @@ public enum DataKind : byte
     [BestFriend]
     internal static class DataKindExtensions
```

Comment:
(nit) - maybe this could be `InternalDataKindExtensions`?

Reply:
Sure.

---
In reply to: [259953370](https://github.com/dotnet/machinelearning/pull/2661#discussion_r259953370) [](ancestors = 259953370)